README.md

```diff
@@ -45,7 +45,9 @@ const appActivity = `${bundleId}.MainActivity`;
 
 const stopApp = () => execSync(`adb shell am force-stop ${bundleId}`);
 const startApp = () =>
-  execSync(`adb shell am start ${bundleId}/${appActivity}`);
+  execSync(
```

Comment:
We could also remove `const appActivity = `${bundleId}.MainActivity`;` from line 44 here

Reply:
Good catch! Done ✅